### PR TITLE
Add Windows 10 Theme

### PR DESCRIPTION
**Pull request by bbmaster123**
Let me know if this looks okay!
Its totally okay if you think we should wait until the issues are fixed before merge
Thanks :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Start Menu Styler mod. The following themes are available:
 | [SideBySide](Themes/SideBySide/README.md) | [![SideBySide](Themes/SideBySide/screenshot-small.png)](Themes/SideBySide/screenshot.png)
 | [SideBySide2](Themes/SideBySide2/README.md) | [![SideBySide2](Themes/SideBySide2/screenshot-small.png)](Themes/SideBySide2/screenshot.png)
 | [SideBySideMinimal](Themes/SideBySideMinimal/README.md) | [![SideBySideMinimal](Themes/SideBySideMinimal/screenshot-small.png)](Themes/SideBySideMinimal/screenshot.png)
+| [Windows10 (WIP)](Themes/Windows10/README.md) | [![Windows10](Themes/Windows10/screenshot-small.png)](Themes/SideBySideMinimal/screenshot.png)
```
Comment (m417z) [inline README.md:57]: Fix the image link (still `SideBySideMinimal`).

Comment (bbmaster123) [reply README.md]: whoops fixed

**Comment by m417z** [comment]
Thanks! BTW are you a member of [the Windhawk Discord channel](https://discord.gg/WZgXScMud7)? I think some people will like your work on this.

**Comment by bbmaster123** [comment]
NP!
I'm not, and for some reason, not sure why but I'm not able to accept the invite, I only really use discord to download stuff haha

**Comment by m417z** [comment]
Not sure why... I created a new temporary link, maybe it will work for you:
https://discord.com/invite/T9GmQA6s

**Comment by bbmaster123** [comment]
that one worked, not sure haha
thanks again!

**Pull request merged**